style.css

```diff
@@ -612,7 +729,7 @@ input[type=submit]:hover {
 	margin: auto auto auto auto;
 }
 
-/* Footer */
+/* Footers */
 
 .footer, .contact_footer  {

```

Comment:
The font used for all three copyright divs work very well, i will consider using it in the final colours and fonts design